wwr/about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="A Rafting Company">
```

Comment:
You are missing meta author. Please add in the head of your html:

![image](https://github.com/user-attachments/assets/83429437-b784-4f04-87fa-6123d1c36f70)

```html
<meta name="author" content="[YOUR NAME]">
```